web_ui_next.py

```diff
@@ -143,9 +147,8 @@ def on_clear_history():
 
 
 with st.chat_message("user"):
-    st.text_area(
-        "Context message", context_message["content"], height=200, disabled=True
-    )
+    st.write("**Cotext message**")
+    st.write(context_message["content"])
```

Comment:
I wanted to keep the context message as markdown, as it makes copying the HTML easier.